### ISSUE TITLE
[ITEM] The Egg of Mortal Essence

### ISSUE THREAD

**Issue by Brayton-mania** [WoWManiaUK/Redemption#5152]
**Links:**
https://www.wow-mania.com/armory/?item=40685 [the trinket]

https://www.wow-mania.com/armory/?spell=48181 [Haunt]
https://www.wow-mania.com/armory/?spell=63108 [Siphon Life]
https://www.wow-mania.com/armory/?spell=47893 [Fel armor]
https://www.wow-mania.com/armory/?spell=47857 [Drain Life]

**What is Happening:**
The trinket is not proccing from any of my warlocks Self healing spells and talents linked above.

**What Should happen:**
It should proc from any healing either direct or over times.

**Comment by Oganku** [comment]
Procs from Drain Life and Health Funnel right now. Note that Drain life is considered a HoT and will only proc it if it actually heals you (overheal will not).

From skimming through wowhead comments and patch notes, I found that it should not proc from Haunt or Siphon Life, and that Fel Armor and Life Tap proccing it were fixed around 3.0.9 and in 3.3.0 respectively. 

TL;DR: No issue here.

https://www.wowhead.com/item=40685/the-egg-of-mortal-essence#comments
![image](https://user-images.githubusercontent.com/34783736/100023414-f0a97c80-2db2-11eb-937b-70ae7df3ab22.png)

https://wow.gamepedia.com/Patch_3.3.0
![image](https://user-images.githubusercontent.com/34783736/100023235-99a3a780-2db2-11eb-9ba5-104743628046.png)
